companion/src/modeledit/customfunctions.cpp

```diff
@@ -507,6 +507,14 @@ void CustomFunctionsPanel::refreshCustomFunction(int i, bool modified)
           cfn.param = (uint8_t)fswtchParamT[i]->currentIndex();
         populateFuncParamCB(fswtchParamT[i], func, cfn.param);
         widgetsMask |= CUSTOM_FUNCTION_SOURCE_PARAM;
+      } else if (func == FuncSetScreen) {
+        fswtchParam[i]->setDecimals(0);
+        fswtchParam[i]->setMinimum(0);
+        fswtchParam[i]->setMaximum(5);  // This assumes that maximum number of custom screen is 5
```

Comment:
Magic numbers! I'd wait until @elecpower comments, but this could probably be put in `companion/src/constants.h` as a constant (perhaps constants for both setMinimum and setMaximum?)

Reply:
The proper solution would be to figure out the actual number of created screens, but I do not see a clean way to do it.

Reply:
Ah, yes... good point... as we won't switch try and switch to a non-existent screen... that can never end well! 😮 😆 Hopefully Neil can say how to get that info as Companion does look at the number of screens configured somewhat for the Custom Screens settings... although in a limited fashion.